dag/main.yml

```diff
@@ -476,6 +476,20 @@ steps:
   data://garden/papers/2023-06-07/commodity_prices:
   - snapshot://papers/2023-06-07/commodity_prices.xlsx
 
+  # Artificial Intelligence Conference Attendance
+  data://meadow/artificial_intelligence/2023-06-14/ai_conferences_total:
+  - snapshot://artificial_intelligence/2023-06-14/ai_conferences_total.csv
+  data://meadow/artificial_intelligence/2023-06-14/ai_small_conferences:
+  - snapshot://artificial_intelligence/2023-06-14/ai_small_conferences.csv
+  data://meadow/artificial_intelligence/2023-06-14/ai_large_conferences:
```

Comment:
Nit - it might be more straightforward to have only a single meadow step `/ai_all_conferences`, load all snapshots there and combine them. Even better would be to skip meadow steps altogether and load all snapshots from the `garden` step (we're not doing anything in meadow step anyway).

Reply:
thanks! :) I wasn't sure if I was allowed to do that but def makes sense haha